sensitivity-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisProvider.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.sensitivity;
+
+import com.powsybl.commons.Versionable;
+import com.powsybl.commons.config.PlatformConfigNamedProvider;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.contingency.ContingenciesProvider;
+import com.powsybl.contingency.EmptyContingencyListProvider;
+import com.powsybl.iidm.network.Network;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Sensitivity analysis provider
+ *
+ * <p>
+ *     Sensitivity analysis is used to assess the impact of a small modification
+ *     of a network variables on the value of network functions.
+ *     This analysis can be assimilated to a partial derivative computed on a given
+ *     network state and on that state modified based on a list of contingencies, if specified.
+ * </p>
+ * <p>
+ *     PTDFs used in Flowbased methodology for example are sensitivity analysis
+ *     results. The sensitivity variables are the GSK shift and the sensitivity function
+ *     are the monitored lines/transformers flows.
+ * </p>
+ * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
+ */
+public interface SensitivityAnalysisProvider extends Versionable, PlatformConfigNamedProvider {
```

Comment:
I thought it could be useful to have both if on implementation side only a simple sensi is implemented and not the systematic one, or only systematic and not simple. But I agree it would make the API a lot easier if there was only method in the interface....
If you still think it's not that useful I will change